qi-doc/scribblings/field-guide.scrbl

```diff
@@ -396,6 +396,8 @@ Yet, either implementation produces the same output: @racket[(list 1 9 25)].
 
 So, to reiterate, while the output of Qi flows will be the same as the output of equivalent Racket expressions, they may nevertheless exhibit a different order of effects.
 
+If you'd like to ensure a specific order of effects, use @racket[effect] at the appropriate points in your flow. If you'd like to use Racket's order of effects, define your flow using @racket[esc] (although this would lose any Qi compiler optimizations).
```

Comment:
What does it mean to "define" a flow "using esc"? I think I understand you to mean to do something like `(define-flow foo (esc …))`, but I'm not sure I understand the connection.

Reply:
"Define" here is intended to mean something like "specify", without the connotation of binding. That is, in `(~> a b (esc (lambda ...)))`, we are considering the third flow in the thread to be defined using Racket instead of defined using Qi. Would "specify" be more clear here, or maybe just "write"?

Reply:
Either of the latter 2 words would be clearer to me; "define" has such a specific meaning in Racket documentation to my ears.